habitica.el

```diff
@@ -509,6 +515,8 @@ TASK is the parsed JSON response."
   (if (< 0 (length (habitica--task-checklist task)))
       (habitica--insert-checklist task))
   (insert "\n")
+  (insert (habitica--task-notes task))
+  (insert "\n")
```

Comment:
if there are no notes for that task is this just going to insert an extra new line? and if so could we prevent that?